lib/node_modules/@stdlib/repl/lib/syntax_highlighter.js

```diff
@@ -82,6 +91,12 @@ function SyntaxHighlighter( repl, ostream ) {
 	// Initialize a buffer to cache the highlighted line:
 	this._highlightedLine = '';
 
+	// Initialize an object storing all available themes:
+	this._themes = THEMES;
+
+	// Initialize a variable storing the current theme:
+	this._theme = '';
```

Comment:
```suggestion
	this._theme = DEFAULT_THEME;
```

This needs to be set to something. Currently, the logic beginning with L118 assumes that a theme has been set externally.